index.html

```diff
@@ -6,6 +6,6 @@
 <body>
 <h2> イベント日時・場所 </h2>
 <p>5月31日 9:30開始</p>
-<p>〇〇〇専門学校〇〇教室</p>
+<p>盛岡情報ビジネス専門学校412教室</p>
```

Comment:
二つの建物がありわかりずらいので盛岡情報ビジネス専門学校ＩＴ　システム間412教室と表記してください
```suggestion
<p>盛岡情報ビジネス専門学校412教室</p>
```